src/python/README.md

```diff
@@ -1,7 +1,8 @@
 # How to Create the Python Wrappers
 
 The wrappers for python are generated using SWIG ([website](http://www.swig.org/)), and for ease of use, the wrappers
-can also be generated with the help of the `distutils` python module. 
+can also be generated with the help of the `distutils` python module.
+On Windows, you may need to add the location where swig.exe was installed to your PATH.
```

Comment:
I'm having installed Visual Studio 2015 on the machine so I have to use an extra step such as calling the command SET VS90COMNTOOLS=%VS140COMNTOOLS% (which temporary sets environmental variable) before executing our script. We need to add an explanation for this too.